generated/nidaqmx/constants.py

```diff
@@ -546,7 +555,7 @@ class ProductCategory(Enum):
     AO_SERIES = 14647  #: AO Series.
     DIGITAL_IO = 14648  #: Digital I/O.
     TIO_SERIES = 14661  #: TIO Series.
-    DSA = 14649  #: Dynamic Signal Acquisition.
+    DYNAMIC_SIGNAL_ACQUISITION = 14649  #: Dynamic Signal Acquisition.
```

Comment:
Good!

Reply:
Updated enum values as in expected constants.py